src/main/java/com/puppycrawl/tools/checkstyle/checks/design/VisibilityModifierCheck.java

```diff
@@ -374,11 +623,11 @@
  * </p>
  * <pre>
  * public class InputPublicImmutable {
```

Comment:
all examples should be same.

please use yours `class MyClass` from above axamples

Reply:
These 2 examples serve a different problem, I think it would be better if I let them independent since other fields in `MyClass` will be a noise. 
If this okay pleas tell me and i'll push the updates for the remain changes you requested.

Reply:
Ok, keep it for now different

Reply:
done